stdparams.go

```diff
@@ -0,0 +1,38 @@
+// +build go1.7
+
+package httprouter
+
+import (
+	"context"
+	"net/http"
+)
+
+type paramsKey struct{}
```

Comment:
The [official example implementation][1] and [documentation][2] for context key types define this to be a type alias for `int`. I doubt there is any actual benefit defining the key like this. There is always only a single instance of this anyway so I suggest to make this as simple and idiomatic as possible.

[1]: https://blog.golang.org/context
[2]: https://godoc.org/context#Context.Value